src/main/java/chess/Piece.java

```diff
@@ -0,0 +1,56 @@
+enum Color {
```

Comment:
Maybe,You can put the enum apart

Reply:
I updated it right now, could you review it please?